src/main/java/com/github/rodionovsasha/commentservice/controllers/CommentController.java

```diff
@@ -0,0 +1,43 @@
+package com.github.rodionovsasha.commentservice.controllers;
+
+import com.github.rodionovsasha.commentservice.entities.Comment;
+import com.github.rodionovsasha.commentservice.exceptions.CommentNotFoundException;
+import com.github.rodionovsasha.commentservice.services.CommentService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+import static com.github.rodionovsasha.commentservice.Application.API_BASE_URL;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@RestController
+@AllArgsConstructor
+@RequestMapping(API_BASE_URL)
+public class CommentController {
+    private static final String COMMENT_BASE_PATH = "/comment";
+    private final CommentService service;
+
+    @GetMapping(value = COMMENT_BASE_PATH + "/{id}", produces = APPLICATION_JSON_VALUE)
+    public Comment getComment(@PathVariable final long id) throws CommentNotFoundException {
```

Comment:
Why do you return `Comment` in `getComment` method but you return `ResponseEntity<Comment>` in `addComment`?

Reply:
So when I do addComment I need to return status code HttpStatus.CREATED (201) instead of OK(200) 
getComment returns aldo comment but with default status 200